spec/spec_helper.rb

```diff
@@ -23,12 +23,114 @@
   end
 end
 
+RSpec.shared_examples "should equal 'r01.05.01'" do
```

Comment:
I think this example group  are same as [2012,4,29 of example group](https://github.com/tomiacannondale/era_ja/blob/master/spec/spec_helper.rb#L230).
Because `2012,4,29` of example group covers all patterns, this example group too.

I think better to delete `2012,4,29` of example group .

Reply:
In my opinion, there is no need to delete `2012,4,29` example groups.
Because these specs are kind of specification of this gem and cares about different era . 
In addition, we still lives in the Heisei era until the end of this April and have chance to deal with datas of the Heisei era even after 5/1. 
So, `Better than nothing` for me.

But it's up to you, though :)

Reply:
OK.
I understood your idea 👍